core/src/test/scala/sbt/librarymanagement/VersionNumberSpec.scala

```diff
@@ -85,7 +85,7 @@ class VersionNumberSpec extends FreeSpec with Matchers with Inside {
   version("2.10.4-20140115-000117-b3a-sources") { v =>
     assertParsesTo(v, Seq(2, 10, 4), Seq("20140115", "000117", "b3a", "sources"), Seq())
     assertCascadesTo(v, Seq("2.10.4-20140115-000117-b3a-sources", "2.10.4", "2.10"))
-    assertIsCompatibleWith(v, "2.0.0", SemVer)
+    assertIsNotCompatibleWith(v, "2.0.0", SemVer)
```

Comment:
According to SemVer 2.10.4-20140115-000117-b3a-sources should be compat with 2.0.0, right?